packages/logger/test/logger.test.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import { spy as Spy } from 'sinon';
+import { Logger, LogLevel } from '../src/logger';
+import { setLogLevel } from '../index';
+import { debug } from 'util';
+
+describe('@firebase/logger', () => {
+  const message = 'Hello there!';
+  let client: Logger;
+  const spies = {
+    debugSpy: null,
+    logSpy: null,
+    infoSpy: null,
+    warnSpy: null,
+    errorSpy: null
+  };
+  /**
+   * Before each test, instantiate a new instance of Logger and establish spies
+   * on all of the console methods so we can assert against them as needed
+   */
+  beforeEach(() => {
+    client = new Logger('@firebase/test-logger');
+
+    spies.logSpy = Spy(console, 'log');
+    spies.infoSpy = Spy(console, 'info');
+    spies.warnSpy = Spy(console, 'warn');
+    spies.errorSpy = Spy(console, 'error');
+  });
+
+  afterEach(() => {
+    spies.logSpy.restore();
+    spies.infoSpy.restore();
+    spies.warnSpy.restore();
+    spies.errorSpy.restore();
+  });
+
+  function testLog(message, channel, shouldLog) {
+    /**
+     * `debug` logs also go through log channel
```

Comment:
This suggests you are currently intentionally merging log and debug messages, but I didn't see this explained anywhere... Did I miss it?

Reply:
I pulled the `beforeEach`. Things should be working as expected.